packages/xs-extra/xapi-xenopsd.master/opam

```diff
@@ -1,29 +1,20 @@
 opam-version: "1.2"
+name: xapi-xenopsd
 maintainer: "xen-api@lists.xen.org"
 authors: "xen-api@lists.xen.org"
 homepage: "https://github.com/xapi-project/xenopsd"
-build: [
-	["./configure"]
-	[make]
-]
-install: [
- 	["./setup.bin" "-install"]
-]
-remove: [
-        ["make" "setup.data"]
-        ["./setup.bin" "-uninstall"]
-        ["ocamlfind" "remove" "xenopsd"]
-]
+build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
+build-test: [[ "jbuilder" "runtest" "-p" name "-j" jobs ]]
 depends: [
-  "oasis" {build}
-  "ocamlfind" {build}
+  "jbuilder" {build}
+  "base"
```

Comment:
Why do we need base? Had we missed this previously? Is this the JaneStreet Base library?

Reply:
I think this initially came in as a dependency of `configurator`, but then we've seen its impossible to install base without upgrading the other janestreet packages, so it should probably be removed for now.

Reply:
Yes, apologies, I forgot to remove it!